tasks/config.yml

```diff
@@ -224,3 +224,61 @@
   notify:
   - restart freeswitch
   - reload nginx
+
+- name: FreeSWITCH dialplan quality
+  replace:
+    path: /opt/freeswitch/conf/dialplan/default/bbb_conference.xml
+    regexp: '<action application="conference" data="$1@.*"/>'
```

Comment:
there are two occurences of this line in the file. Do you want to replace just the first or both?

Reply:
The regex is currently not working for me. You need to escape the `$` and `/`.

Reply:
both

Reply:
yes, both.